app.rb

```diff
@@ -146,11 +148,20 @@ def authorization_url(ial:, aal: nil)
         state: random_value,
         nonce: random_value,
         prompt: 'select_account',
-      }.to_query
+        inherited_proofing_auth: params['ip_auth_option'] ? VA_TEST_AUTH_CODE : nil,
+      }.compact.to_query
 
       "#{endpoint}?#{request_params}"
     end
 
+    def user_options
```

Comment:
as of the changes in ab0d1fc25f4a0a143ee227f042fb46dcb3f6ddcb, we can remove this whole method now right?